modules/ROOT/pages/oauth-dance-about.adoc

```diff
@@ -3,35 +3,43 @@ ifndef::env-site,env-github[]
 include::_attributes.adoc[]
 endif::[]
 
-The authentication process performed by the Mule OAuth 2.0 provider, API, and client application conforms to RFC 6749. This process is called the OAuth Dance.
+The OAuth Dance is the authentication process performed by the Mule OAuth 2.0 provider, API, and client application. This authenticatio process conforms to RFC 6749.
 
+== OAuth Dance Authentication Flow
+
+The OAuth Dance authentication flow is illustrated in the following screenshot:
 image::mule-oauth-provider-dance.png[oauth+policy1,height=366,width=640]
 
-1. A token is requested to the provider, specifying client application credentials as needed (the credentials correspond to a Client ID and secret generated by API Platform).
-2. Mule OAuth 2.0 provider returns a token.
-3. An HTTP request is performed against the protected resource (ie resource where the OAuth 2.0 Access Token Enforcement policy is applied), including the token either as an authentication header or a query parameter.
-- The protected resource *must* be linked with the API definition through Autodiscovery.
-4. The OAuth 2.0 Access Token Enforcement policy intercepts this request and communicates with the Provider to validate the token.
-5. If the token is correctly validated:
-- The token is whitelisted and kept on record until expiration. Any further requests that contain this token are not validated against the OAuth provider.
-- The associated client ID of the client application credentials is checked to make sure that the application has an active contract with the API. (Not Applicable to OpenAM)
-6. The request is forwarded to the resource.
-7. An HTTP response is generated.
+. A token request is sent to the provider, specifying client application credentials as needed. The credentials correspond to a client ID and secret generated by API Platform.
```

Comment:
Numbers here are critical, since correspond to the ones in the provided graphic.

Reply:
I added automatic numbering. Doesn't it render that way?

Reply:
I see numbered steps (1 through 7), with Roman numerals as substeps (i., ii., ...).
I believe that Mysti is working on getting the substeps to appear as lowercase letters (a., b., c., ...)
But the numbering and association indentations are clear.

Reply:
I see them now! :-)

Reply:
Just a comment: Beware of automatic numbering. Just adding some new element to the ordered list, will make the associated diagram unreadable.

Reply:
Ok Thanks!